### DISCUSSION TITLE
how to set multiple webhooks?

### DISCUSSION THREAD

**Discussion by flyingstarlai** [bukhalo/nestjs-telegraf#574] [Q&A] [answered]
the default hookPath is '/secret-path', how to add new multiple hookPath?

**Comment by felinto-dev** [comment, marked as answer]
I don't think that Telegram BOT API Server has support sending updates queries for multiple webhooks URLs. Did you see any documentation that talks about it?

**Comment by evilsprut** [comment]
Telegram Bot API doesn't support it, neither does `telegraf`, and I don't see the point of supporting it in a library either.
